src/client/src/tscUtil.c

```diff
@@ -3035,6 +3035,12 @@ int32_t tscValidateName(SStrToken* pToken, bool escapeEnabled, bool *dbIncluded)
       }
     }
 
+    if (escapeEnabled && pToken->type == TK_ID) {
```

Comment:
For some cases, e.g. "select * from DB.tbl;", the table name is prefixed with its DB NAME as the first part. The database name might be backward quoted, e.g. words in capital cases or Chinese characters, which should be subjected to stringProcess() too.